packages/contracts/contracts/proof-of-use/groups/UserGroups.sol

```diff
@@ -30,6 +30,13 @@ contract UserGroups is OffChainOwnable, TokenTelemetryI, UnionFind {
   {
   }
 
+  function useProven(address who) external view returns (uint usedTokens, uint totalTokens) {
```

Comment:
I think this function could have a slightly clearer name. `usedAndTotalTokensForUser` or something